ms2query/create_library_files.py

```diff
@@ -10,50 +11,48 @@
 from ms2query.create_sqlite_database import make_sqlfile_wrapper
 from ms2query.spectrum_processing import minimal_processing_multiple_spectra, \
     create_spectrum_documents
-from ms2query.app_helpers import load_pickled_file
 
 
 class CreateFilesForLibrary:
     def __init__(self,
                  pickled_spectra_file_name: str,
+                 output_file_sqlite: str,
```

Comment:
We could even decide to call this `output_base_filename` and remove the options to name files individually.

Reply:
I changed this and added that to this pull request